internal/provider/resource_gitlab_user.go

```diff
@@ -74,11 +74,15 @@ var _ = registerResource("gitlab_user", func() *schema.Resource {
 				Default:     false,
 			},
 			"skip_confirmation": {
-				Description: "Boolean, defaults to true. Whether to skip confirmation.",
-				Type:        schema.TypeBool,
-				Optional:    true,
-				Default:     true,
-				ForceNew:    true,
+				Description: `Boolean, defaults to true. Whether to skip confirmation.
+				
```

Comment:
The rendering of this is not really nice:

![image](https://user-images.githubusercontent.com/1008252/190596873-1c0f57cb-3a99-4bdf-9284-14a52a4fe630.png)

Do you mind removing the blank line. It would change it to:

![image](https://user-images.githubusercontent.com/1008252/190596994-84111968-a4f7-422a-9da9-8d41ce183c82.png)

```suggestion
```

Reply:
@PatrickRice-KSC

Reply:
Wow, I'm not sure how I missed this comment for over a month. Jeez, sorry guys (and thanks for the reminder @Shocktrooper !)

I'll get those changes made so we can get this merged 😄

Reply:
No worries 😄 I was just like huh I don't think there's anything else to do besides this review comment